### ISSUE TITLE
Convert repo to use Markdown

### ISSUE THREAD

**Issue by adrianmcli** [kennetpostigo/anIntroductionToReason#6]
@kennetpostigo I wouldn't mind helping you do this, but I'm not sure where to start. Can you give me a direction?

**Comment by kennetpostigo** [comment]
Hey @adrianmcli  , so the goal is to move everything over to gatsby. Which will allow us to use actual markdown but still be able to use react and mix both flavors together markdown for static content and use react components for dynamic content. The main blocker is that gatsby 1.0 is not done yet. 1.0 is an overhaul and large improvement over the current gatsby. It supports many of the features I am looking for:
1. Easy and quick editing
2. Offline reading
3. Code splitting routes for small payloads
4. Really fast pageloads
5. Easy interop with react (maybe even rehydrate)
6. Easier format for people to edit and add to the guide.

We can currently use gatsby 1.0 if you `npm install -g gatsby@next` however you would have to fork gatsbygram to use it and strip it down to a basic state that we can build on. I just haven't had the time to do this because of school and work. So using gatsby would be the start. I have all the content on gitbook.io in markdown format currently so it would literally be copy and paste when transitioning to gatsby and making 1.0 work for us is the main work that needs to be done.

**Comment by adrianmcli** [comment]
Interesting. I can certainly try to strip down Gatsbygram. I suppose you are talking about this: https://github.com/saiikunst/gatsbyjs/tree/1.0/examples/gatsbygram

My only other question is whether or not we want to do this now, since they are still in alpha (I'd feel more comfortable if it was beta). Since you have all the Markdown ready-to-go, maybe my time would be better used going through the existing docs and marking typos and other issues.

**Comment by kennetpostigo** [comment]
@adrianmcli yea thats the one.

Yea definitely you can do that as well, just give me your email and I'll give you write permissions on the gitbook. Make sure you have a gitbook account tho

**Comment by adrianmcli** [comment]
@kennetpostigo sure, the email under my GH profile is tied to a gitbook account.

**Comment by kennetpostigo** [comment]
check your email when you get a chance

**Comment by kennetpostigo** [comment]
@adrianmcli hey so rdn has been made public it's not done but you can contribute to the content and functionality. There are issues open that are blockers to it going online. Let me know if you are still interested in helping out http://github.com/kennetpostigo/rdn

**Comment by adrianmcli** [comment]
Yes! I'm still very interested. I'll be taking a look at the issues list
later today.
